source/ed/build_helper.h

```diff
@@ -134,6 +135,10 @@ struct VJ {
         _route_name = r;
         return *this;
     }
+    VJ& direction_type(const std::string& d) {
```

Comment:
This must be associated to route's build (above), not to VJ.
As it won't trigger a route build for "backward" if there is already one for "forward" (which is what one would hope using it).

Reply:
You have right, it is more clear to link the **direction_type** to the _route_ with a function like this ?
```
ed::builder b("20190101");
 b.route("route_name", "direction_type");
```